CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 * Leverage API supported sorting of columns on pre-registrations records list. Refs UIU-3249.
 * Fix issue with `Proxy borrower` field value. Refs UIU-3290.
 * Remove duplicates from the keyboard shortcut modal. Refs UIU-3026.
+* Create/Edit a Patron Block - HTML page title. Refs UIU-3302.
```

Comment:
Could you please reword this to "Add HTML page title to add/edit patron block page"?